cypress/e2e/views/common.view.ts

```diff
@@ -52,3 +52,5 @@ export const divBottom = "#tags-pagination-bottom";
 export const selectFilter = "div.pf-c-toolbar__group.pf-m-toggle-group.pf-m-filter-group.pf-m-show";
 export const itemsSelectInsideDialog = "div[role='dialog'] button[aria-label='Select']";
 export const stakeHoldersTable = "table[aria-label='Stakeholders table']";
+export const nameHelperBusiness = "#business-service-name-helper";
+export const nameHelperStakeholderGroup = "#-helper";
```

Comment:
I don't think devs did this on purpose, probably this id will change in new builds.